mailbox/tika/src/main/java/org/apache/james/mailbox/tika/TikaConfiguration.java

```diff
@@ -58,24 +64,52 @@ public Builder timeoutInMillis(int timeoutInMillis) {
             return this;
         }
 
+        public Builder cacheEvictionPeriod(Duration duration) {
+            this.cacheEvictionPeriod = Optional.of(duration);
+            return this;
+        }
+
+        public Builder cacheEvictionPeriod(Optional<Duration> duration) {
+            this.cacheEvictionPeriod = duration;
+            return this;
+        }
+
+        public Builder cacheWeightInBytes(long weight) {
+            this.cacheWeightInBytes = Optional.of(weight);
+            return this;
+        }
+
+        public Builder cacheWeightInBytes(Optional<Long> weight) {
+            this.cacheWeightInBytes = weight;
+            return this;
+        }
+
         public TikaConfiguration build() {
             Preconditions.checkState(host.isPresent(), "'host' is mandatory");
             Preconditions.checkState(port.isPresent(), "'port' is mandatory");
             Preconditions.checkState(timeoutInMillis.isPresent(), "'timeoutInMillis' is mandatory");
             Port.assertValid(port.get());
 
-            return new TikaConfiguration(host.get(), port.get(), timeoutInMillis.get());
+            return new TikaConfiguration(host.get(), port.get(), timeoutInMillis.get(),
+                cacheEvictionPeriod.orElse(DEFAULT_CACHE_EVICTION_PERIOD),
```

Comment:
how I can disable the cache ?

Reply:
Do you want a `tika.cache.enable=true|false` or saying "the weight set to 0" in the doc is OK to you?

Reply:
`tika.cache.enable=true|false`